config/jobs/cert-manager/OWNERS

```diff
@@ -1,5 +1,6 @@
 approvers:
 - munnerz
 - meyskens
+- wallrj
```

Comment:
Be good to add the CM team here so that those within the team can approve changes on behalf of CM

Reply:
I added individuals, but can I use @jetstack/team-cert-manager  instead?

Reply:
Not sure if that's possible right now  - https://github.com/kubernetes/test-infra/issues/10065